### PR TITLE
Am product model

### PR DESCRIPTION
**Pull request by AymenMech**

### CODE REVIEW

**Comment by florentdab** [comment]
It's ok for me,
Good job

File: app/Resources/SyliusShopBundle/views/Product/show.html.twig [hunk 1]
```diff
@@ -51,9 +51,52 @@
...
+                            <img src="{{ asset("media/image/"~ article.path) }}" class="img_book" alt="{{ article.code }}"  data-toggle="modal" data-target="#{{ article.id }}">
+                            <div class="modal fade" id="{{ article.id }}" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+                                <div class="modal-dialog" role="document">
+                                    <div class="modal-content">
+                                        <div class="modal-header">
+                                            <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                                            <h4 class="modal-title" id="myModalLabel">{{ article.code }}</h4>
+                                        </div>
+                                        <div class="modal-body">
+                                            <img src="{{ asset("media/image/"~ article.path) }}">
```
Comment (2binfree) [inline app/Resources/SyliusShopBundle/views/Product/show.html.twig]: this link is ok ?

Comment (AymenMech) [inline app/Resources/SyliusShopBundle/views/Product/show.html.twig]: I know that looks weird but it works =)

**Comment by FullFrontal92** [comment]
Ok for me

**Pull request merged**